LayoutTests/http/tests/site-isolation/mouse-events/resources/context-menu-event-listener.html

```diff
@@ -0,0 +1,5 @@
+<script>
+addEventListener("contextmenu", () => {
+    window.parent.postMessage("", "*"); 
```

Comment:
It would be nice to send a string then check it in the main frame event listener.